src/ScriptEditor/ScriptEditor.ts

```diff
@@ -100,6 +103,25 @@ export class ScriptEditor {
           2686,
         ],
       });
+      // Sync all javascript and typescript text models to both language servers.
+      {
+        // `monaco.languages.typescript.get{Java,Type}ScriptWorker` returns a promise that
+        // fires with a function that takes a list of `monaco.Uri`s and sync's them with the
+        // worker. (It also returns the worker, but we don't care about that.)
+        const languageWorker = new Promise<(...uris: monaco.Uri[]) => unknown>((resolve) =>
+          monaco.languages.onLanguage(language, () => getLanguageWorker().then(resolve)),
+        );
+        // Whenever a model is created, arange for it synced to the language server.
+        monaco.editor.onDidCreateModel((model) => {
+          if (language === "typescript" && isLegacyScript(model.uri.path)) {
```

Comment:
I think we don't want *either* type synced? IIRC, .script files can only import other .script files.

Reply:
Ideally we'd have a custom language for script files, but we tell monaco that they are `javascript`, so it will get synced to the javascript worker at some point eventually anyway.

We could probably get away with just syncing to the opposite language server (so when `language != model.getLanguageId()`), but it doesn't hurt to tell them to sync an extra time, and I haven't dug into the timing of when they get synced, and I think it is reasonable to eargerly sync them if they aren't already.